sql/core/src/test/resources/sql-tests/results/udf/udf-inline-table.sql.out

```diff
@@ -0,0 +1,153 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 17
+
+
+-- !query 0
+select udf(col1), udf(col2) from values ("one", 1)
+-- !query 0 schema
+struct<CAST(udf(cast(col1 as string)) AS STRING):string,CAST(udf(cast(col2 as string)) AS INT):int>
+-- !query 0 output
+one	1
+
+
+-- !query 1
+select udf(col1), udf(udf(col2)) from values ("one", 1) as data
+-- !query 1 schema
+struct<CAST(udf(cast(col1 as string)) AS STRING):string,CAST(udf(cast(cast(udf(cast(col2 as string)) as int) as string)) AS INT):int>
+-- !query 1 output
+one	1
+
+
+-- !query 2
+select udf(a), b from values ("one", 1) as data(a, b)
+-- !query 2 schema
+struct<CAST(udf(cast(a as string)) AS STRING):string,b:int>
+-- !query 2 output
+one	1
+
+
+-- !query 3
+select udf(a) from values 1, 2, 3 as data(a)
+-- !query 3 schema
+struct<CAST(udf(cast(a as string)) AS INT):int>
+-- !query 3 output
+1
+2
+3
+
+
+-- !query 4
+select udf(a), b from values ("one", 1), ("two", 2), ("three", null) as data(a, b)
+-- !query 4 schema
+struct<CAST(udf(cast(a as string)) AS STRING):string,b:int>
+-- !query 4 output
+one	1
+three	NULL
+two	2
+
+
+-- !query 5
+select a, udf(b) from values ("one", null), ("two", null) as data(a, b)
+-- !query 5 schema
+struct<>
+-- !query 5 output
+org.apache.spark.sql.AnalysisException
+cannot resolve 'CAST(udf(cast(b as string)) AS NULL)' due to data type mismatch: cannot cast string to null; line 1 pos 10
```

Comment:
@imback82, let's comment out this test with a comment saying the current UDF is not supported for the conversion from string to null for now.

Reply:
@HyukjinKwon Do you prefer that or just work around with `udf(a), b`?

Reply:
Hm, yea let's do that.

Reply:
OK, fixed. New commit pushed.